### DISCUSSION TITLE
What to consider when analysing formalin fixed samples

### DISCUSSION THREAD

**Discussion by FlorianProdinger** [vdemichev/DiaNN#517] [Q&A] [answered]
Hello everone,

I am wondering whether somebody in this forum has experience with FFPE samples and DIA-NN.
My lab is currently preparing to measure parafin embedded formalin fixed (FFPE) tissue samples and I was asked to look up whether DIA-NN has a settings that support this analysis. Specifically, a feature to analyse methylated Lysin would be appreciated.

I have checked the github page and the manual, if I have missed a part of the Documentation about FFPE, please refer me to it.

If the authors see this post: Thanks for developing DIA-NN. It's a great tool!
Thank you!

**Comment by vdemichev** [comment, marked as answer]
Hi Florian,

We haven't specifically tested this for FFPE, but in general adding modifications is simple in DIA-NN, e.g. just add to 'Additional options':
--var-mod UniMod:34,14.015650,K
--strip-unknown-mods

Best,
Vadim

**Reply by FlorianProdinger** [reply]
Hello vdemichev!

Thanks so much for your quick reply! I thought that it'll work by just using ptm related options, but i thought I'd better ask if somebody has experience.
Again, I'm super grateful for the software and your reply! I'll post how well it went after we tried it out!

Have a great day,
Florian
